app/views/organizations/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1 class="page-header">
+  <%= @organization.login %>
+</h1>
+
+<%= link_to 'Remove Classroom', @organization, class: 'btn btn-sm btn-danger', role: 'button',
```

Comment:
As or right now all it does is remove the `Classroom` from the DB and remove the relation to the `User`. But that could change in the future. I'm going to move this into the `User` settings page and give some more context.